alcs-frontend/src/app/features/application/decision/decision-v2/decision-v2.component.html

```diff
@@ -118,8 +118,13 @@ <h3>Condition Compliance</h3>
             </div>
 
             <div>
-              <div class="subheading2">Subject To Conditions</div>
-              {{ decision.isSubjectToConditions ? 'Yes' : 'No' }}
+              <div *ngIf="!!decision.modifies || !!decision.reconsiders" class="subheading2">
+                <span *ngIf="!!decision.modifies">Modification Outcome</span>
+                <span *ngIf="!!decision.reconsiders">Reconsideration Outcome</span>
```

Comment:
They are arrays, so empty arrays will be falsy and an array with at least one item is truthy.

Reply:
I just checked on node repl and it returns`!![]` as truthy, do you want to test and confirm that an empty array would be falsey? Otherwise, I would suggest checking the array length to be safe.

Reply:
Just checked, they are either objects or null. The check is for them to not be null.